internal/characters/kaveh/cons.go

```diff
@@ -43,10 +43,13 @@ func (c *char) c4() {
 	c.AddReactBonusMod(character.ReactBonusMod{
 		Base: modifier.NewBaseWithHitlag("kaveh-c4", -1),
 		Amount: func(ai combat.AttackInfo) (float64, bool) {
-			if ai.AttackTag == attacks.AttackTagBloom {
-				return 0.6, false
+			if ai.AttackTag != attacks.AttackTagBloom {
+				return 0, false
 			}
-			return 0, false
+			if ai.ActorIndex != c.Index {
```

Comment:
This `ActorIndex` check seems unnecessary. When would this be called for characters other than Kaveh himself?